src/Resources/config/actions.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Sonata\AdminBundle\Action\AppendFormFieldElementAction;
+use Sonata\AdminBundle\Action\DashboardAction;
+use Sonata\AdminBundle\Action\GetShortObjectDescriptionAction;
+use Sonata\AdminBundle\Action\RetrieveAutocompleteItemsAction;
+use Sonata\AdminBundle\Action\RetrieveFormFieldElementAction;
+use Sonata\AdminBundle\Action\SearchAction;
+use Sonata\AdminBundle\Action\SetObjectFieldValueAction;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\DependencyInjection\Reference;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->services()
+
+        ->set(DashboardAction::class, DashboardAction::class)
+            ->public()
+            ->args([
+                '%sonata.admin.configuration.dashboard_blocks%',
+                new Reference('sonata.admin.breadcrumbs_builder'),
```

Comment:
why dont you use `ref` method?

Reply:
[It is deprecated since 5.1](https://github.com/symfony/symfony/blob/5.1/src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php#L86). I started using it, but then I realised of that.

Reply:
then it would be nice to add a comment to replace all those news with service function once php < 5.1 is drop

Reply:
Also, this should be a ReferenceConfigurator if you are not using the helper functions.